<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>731</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="nextBtn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>510</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="nameTxt">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="nameLbl">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservation Name:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Room Size</string>
      </property>
      <widget class="QRadioButton" name="kingBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1-King</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="queenBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>2-Queen</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="viewBtns">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>View Type</string>
      </property>
      <widget class="QRadioButton" name="standardBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="atriumBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Atrium</string>
       </property>
      </widget>
     </widget>
     <widget class="QSpinBox" name="numAdultsBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="numChildBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="numAdultsLbl">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>280</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Num Adults:</string>
      </property>
     </widget>
     <widget class="QLabel" name="numChildrenLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Num children</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="parkingBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>380</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking</string>
      </property>
     </widget>
     <widget class="QLabel" name="costLbl">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>470</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cost:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="numDays">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="numDaysLbl">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Num Days:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="totalCostLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Total cost: </string>
      </property>
     </widget>
     <widget class="QLabel" name="roomCostLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Room cost:</string>
      </property>
     </widget>
     <widget class="QLabel" name="taxCostLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Tax cost:</string>
      </property>
     </widget>
     <widget class="QLabel" name="parkingCostLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking cost:</string>
      </property>
     </widget>
     <widget class="QLabel" name="resortFeeCostLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Resort fee cost:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="visaRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>visa</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="masterCardRadioButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>mastercard</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="discoverRadioButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>discover</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="americanExpressRadioButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>320</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>American Express</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="creditCardLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="cardExperationDate">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="cardExperationDateLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>360</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Experation date:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="payButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>520</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
